.github/workflows/go.yml

```diff
@@ -47,7 +47,8 @@ jobs:
         echo "Coverage output is ${{ steps.test.outputs.coverage }}"
     
     - name: Update coverage badge
-      if: github.ref == 'refs/heads/master'
+      # The dash at the end makes this to never run, while we set up the ODH badge.
+      if: github.ref == 'refs/heads/master-'
```

Comment:
I would prefer to add a `if: false` to disable it instead of a wrong parameter
See https://stackoverflow.com/a/75905933
Please add a comment next to the if with the reason of the conf (and/or a link to a ticket to track how we plan to solve it)

Reply:
Fixed!
Thanks for the suggestion.